src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -50,8 +55,8 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
         </View>
         <View style={styles.containerButton}>
           <Pressable style={styles.followAction(32, 32)}>
-            <Animated.View style={{position: 'absolute', opacity: followIconFadeAnimation}}>
-              <MemoIc_Checklist width={32} height={32} />
+            <Animated.View style={{position: 'absolute', top: 4, opacity: followIconFadeAnimation}}>
+              <MemoIc_Checklist width={32} height={32} color={isAnon ? '#107793' : '#4782D7'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIc_Checklist` component is now being set conditionally based on the `isAnon` prop. If `isAnon` is true, then the color will be `'#107793'`. Otherwise, it will be `'#4782D7'`. This change seems to be correct and doesn't introduce any issues.
```diff
- <MemoIc_Checklist width={32} height={32} />
+ <MemoIc_Checklist width={32} height={32} color={isAnon ? '#107793' : '#4782D7'} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->